system/jlib/jutil.cpp

```diff
@@ -3349,7 +3349,7 @@ const char * generatePassword(StringBuffer &pwd, int pwdLen)
     const char alphaUC[] = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
     const char alphaLC[] = "abcdefghijklmnopqrstuvwxyz";
     const char numeric[] = "0123456789";
-    const char symbol[] = "~@#$%^*()_-+={[}]:,.?";
+    const char symbol[] = "~@#%^*_-+{[}]:,.?";
 
     //Ensures each character group used at least once
     srand(time(0));
```

Comment:
Not new, but this doesn't feel like a secure way to generate passwords to me... If you know the approximate date that an account was created you can brute force through the passwords that would have been generated on that date very easily...